atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CassandraRepairHelper.java

```diff
@@ -80,7 +81,8 @@ private static void onRemoval(
             AtlasService atlasService,
             ReloadingCloseableContainer<CqlCluster> reloadingCloseableContainer,
             RemovalCause _removalCause) {
-        log.info("Closing cql cluster container", SafeArg.of("atlasService", atlasService));
+        Exception exceptionJustForLogging = new SafeRuntimeException("I exist to show you the stack trace");
+        log.info("Closing cql cluster container", SafeArg.of("atlasService", atlasService), exceptionJustForLogging);
```

Comment:
actually this one was needed on onClose of cluster because we know the onRemoval callback is called from cache only

Reply:
sorry i meant we should move the log to inside `reloadingCloseableContainer.close();`